### DISCUSSION TITLE
Requests by organization

### DISCUSSION THREAD

**Discussion by vincent-nagy** [cloudbees/zendesk-java-client#481] [Q&A] [answered]
I noticed there is a call for /organizations/{{organizationId}}/tickets.json but not one for /organizations/{{organizationId}}/requests.json.

Am I missing it or hasn't it been added yet?

The use case is showing all requests on behalf of the user that belongs to their organization.

**Comment by aheritier** [comment]
Effectively it's not existing. I am adding it.

**Comment by aheritier** [comment, marked as answer]
Fixed in 0.17.0 ( #487 )
